### ISSUE TITLE
Not able to set custom configs

### ISSUE THREAD

**Issue by ahmadalibaloch** [verdan/flaskoidc#31]
I am getting CONFIG_URL is not set error.  Can you guide how to setup? I am trying to setup in [superset](https://github.com/apache/superset)

<img width="971" alt="image" src="https://user-images.githubusercontent.com/1916642/173448971-cf05d697-ae57-4aca-afa5-505246bd4004.png">

**Comment by verdan** [comment]
yes, I believe there is a bug. I will make sure to fix this in the next release. 
In the meantime, can you please try to use the environment variables instead and let me know if that unblocks you?
